### DISCUSSION TITLE
Tautology and Falsum symbols?

### DISCUSSION THREAD

**Discussion by SUPERCILEX** [typst/typst#2564] [General]
### Description

Falsum is technically present as `perp`, but I can't find taut.

### Use Case

Logic.

**Comment by Enter-tainer** [comment]
top

**Comment by matthew-e-brown** [comment]
`perp` is U+27C2, which _really is_ "perpendicular". For a falsum, you want U+22A5, the ["up tack"](https://en.wikipedia.org/wiki/Up_tack), which is available as `bot` or `tack.t`. Similarly, for a tee/verum, you want [U+22A4 `DOWN TACK`](https://en.wikipedia.org/wiki/Tee_(symbol)), available in Typst as `top` or `tack.b`.

(see also `tack.b.short` and `tack.t.short`).

**Comment by SUPERCILEX** [comment]
Neat! Do you know how the website does search aliases? Seems like all we need to do is add some

**Comment by matthew-e-brown** [comment]
I don't... but I definitely agree that the search can be a little unwieldy at times. Searching for 'tack' gives braces, card suits, triangles, squares, and arrows all before the actual tacks way at the bottom. I wonder if there's an open issue for symbols' search yet. 🤔

**Comment by laurmaedje** [comment]
The search uses fuse.js and it just isn't really good in my experience so far.
